javascript/ql/src/Vue/ArrowMethodOnVueInstance.qhelp

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE qhelp PUBLIC
+"-//Semmle//qhelp//EN"
+"qhelp.dtd">
+<qhelp>
+
+<overview>
+	<p>
+
+		The Vue framework invokes the methods of a Vue instance with the
+		instance as the receiver.  This binding of method and instance is
+		however not possible to do for arrow functions, so the
```

Comment:
The "this binding of method and instance is however not possible to do" bit doesn't read well. 
Possibly rephrase or replace "to do" with another verb. Why can't you implement this binding for arrow functions?